rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -260,3 +273,17 @@ func (r *Layer1Relayer) shouldUpdateGasOracle(baseFee uint64, blobBaseFee uint64
 
 	return false
 }
+
+func (r *Layer1Relayer) commitBatchReachTimeout() bool {
+	fields := map[string]interface{}{
+		"rollup_status IN ?": []types.RollupStatus{types.RollupCommitted, types.RollupFinalizing, types.RollupFinalized},
+	}
+	orderByList := []string{"index DESC"}
```

Comment:
it's better put the sql to batch orm file.

Reply:
it's just preparing some args for `batchOrm.GetBatches`.  It is used same way in other places